packages/store/src/internal/internals.ts

```diff
@@ -40,15 +40,21 @@ export interface MetaDataModel {
   selectFromAppState: SelectFromState | null;
   children?: StateClass[];
   instance: any;
+  sharedSelectorOptions?: InternalSelectorOptions;
 }
 
 export type SelectFromState = (state: any) => any;
 
+export interface InternalSelectorOptions {
```

Comment:
why not give a different name?

`SharedSelectorOptions`

Reply:
I have kept the name `InternalSelectorOptions` to keep things consistent. They should not be part of the public API and this needs to be explicit.